packages/app/src/GoogleBtn.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import { GoogleLogin, GoogleLogout } from "react-google-login";
+
+const CLIENT_ID = "216565480006-kir74928iff14l4ln6p5kn0llrd5jk9a.apps.googleusercontent.com";
```

Comment:
If this is going to be a secret specific to KSF, I would recommend adding this to the .env file and read from there instead of hardcoding this into the component. 

You can reference how this has been done in https://github.com/the-difference-engine/ksf/blob/develop/packages/api/helper/sequelize.js